ERROR - 2023-09-27 13:42:22 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in C:\xampp\htdocs\bulut\system\Database\MySQLi\Connection.php:170
Stack trace:
#0 C:\xampp\htdocs\bulut\system\Database\MySQLi\Connection.php(170): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bulut', 3306, '', 0)
#1 C:\xampp\htdocs\bulut\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\bulut\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\bulut\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\bulut\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\bulut\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\bulut\app\Controllers\auth.php(39): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\bulut\system\CodeIgniter.php(927): App\Controllers\auth->login()
#8 C:\xampp\htdocs\bulut\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#9 C:\xampp\htdocs\bulut\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\bulut\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in C:\xampp\htdocs\bulut\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\bulut\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\bulut\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\bulut\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\bulut\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\bulut\system\BaseModel.php(623): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\bulut\app\Controllers\auth.php(39): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\bulut\system\CodeIgniter.php(927): App\Controllers\auth->login()
#7 C:\xampp\htdocs\bulut\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#8 C:\xampp\htdocs\bulut\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\bulut\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-09-27 13:42:22 --> Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_users`
WHERE `EMAIL` = :EMAIL:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(623): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\auth.php(39): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\auth->login()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
